app/deed/sign/server.py

```diff
@@ -0,0 +1,20 @@
+from time import strftime
+from flask import request, jsonify, abort
+
+
+def register_routes(blueprint, deed_api):
+    @blueprint.route('/deed/<deed_id>/<borrower_id>/signature',
+                     methods=['POST'])
+    def sign_deed(deed_id, borrower_id):
+        borrower_name = request.form['borrower_name']
+
+        def create_signature():
+            return borrower_name + "_" + strftime("%d/%m/%Y_%H:%M:%S")
```

Comment:
This signature is a temporary solution, therefore date format doesn't matter at all.